src/View/Output.js

```diff
@@ -0,0 +1,32 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { RANK } from "../constants/rules.js";
+import { OUTPUT_MESSAGES } from "../constants/message.js";
+
+/** 출력과 관련된 로직들을 관리하는 class입니다. */
+class Output {
+  /** 구매 금액에 따른 티켓 개수를 출력합니다. */
+  outputTicketAmount(amount) {
+    MissionUtils.Console.print(OUTPUT_MESSAGES.PURCHASE(amount));
+  }
+
+  /** 각 티켓의 로또 번호를 출력합니다. */
+  outputTickets(tickets) {
+    for (let i = 0; i < tickets.length; i++) {
+      const ticketString = tickets[i].join(", ");
+      MissionUtils.Console.print(`[${ticketString}]`);
+    }
+  }
+
+  /** 당첨 통계와 수익률을 출력합니다. */
+  printResult(resultArr, profitRate) {
+    MissionUtils.Console.print(OUTPUT_MESSAGES.RESULT_STATEMENT);
+    for (let rank = 5; rank > 0; rank -= 1) {
```

Comment:
```suggestion
    for (let rank = 5; rank > 0; rank --) {
```
코드를 보면서 다른곳은 for문에서 ++이런식으로 하시는데 여기는 -=이여서 혹시 --와 같은 동작을 한다면 
--로 통일은 어떨까요? 코드를 봤을때 대략 --가 될거같은데 제가 저렇게는 써보지않아서 조심스럽게 말씀드립니다 ㅎ

Reply:
엇 그렇네요! 통일성에 도움이 많이 될 것 같습니당! 감사합니다 ☺️ 👍